test/ui-test/testSuites/suite_status/shared/steps/communitySteps.py

```diff
@@ -36,6 +36,14 @@ def step(context, new_community_channel_name):
 @Then("the user lands on the community channel named |any|")
 def step(context, community_channel_name): 
     _statusCommunityScreen.verify_channel_name(community_channel_name)
+    
+@When("the admin creates a community category named |any|, with channel |any| and with the method |any|")
+def step(context, community_category_name, community_channel_name, method):
+    _statusCommunityScreen.create_community_category(community_category_name, community_channel_name, method)
+    
+@Then("the user lands on the community category named |any|")
```

Comment:
This is not a good name for this test, because it doesn't actually check that we landed on the category, it just checks that the category is created, which is fine. Just rename it something like
`the category named |any| is created`

Reply:
fixed